ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -391,14 +391,15 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
-				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
@nain93 Can you explain any reason changing this?

Reply:
There are cases where 'FLUTTER_BUILD_NUMBER' is not fetched when building.

<img width="1073" alt="스크린샷 2022-12-11 오후 12 35 04" src="https://user-images.githubusercontent.com/73378472/206885091-9d8a651c-649b-46ca-b34e-5044ec2874e3.png">

Reply:
Looks like related to https://stackoverflow.com/questions/65277520/flutter-not-using-build-number-from-pubspec-yaml

It does not look good on setting versions separately on each platform.
Hope we can handle this in the future~